### PR TITLE
propcheck: check for suspicious paths

### PR DESCRIPTION
**Pull request by dkm**
Checks that paths in .amazon.properties are starting with

/opt/compiler-explorer/.

Also rename cppx_blue default config file (was .default. instead of .defaults.).

Signed-off-by: Marc Poulhiès <dkm@kataplop.net>

### CODE REVIEW

File: etc/scripts/util/propscheck.py [hunk 1]
```diff
@@ -94,7 +94,10 @@ def process_file(file: str):
     seen_lines = set()
     duplicate_lines = set()
 
-    disabled = set()
+    suspicious_path = set()
```
Comment (RubenRBS) [inline etc/scripts/util/propscheck.py:97]: Nice idea!

File: etc/scripts/util/test/cases/suspicious_path.properties [hunk 2]
```diff
@@ -0,0 +1,3 @@
+compilers=a:b
+compiler.a.exe=/opt/compiler-explorer/some/bin/gcc
+compiler.b.exe=/wrong/path/bin/gcc
```
Comment (RubenRBS) [inline etc/scripts/util/test/cases/suspicious_path.properties:3]: Newline at EOF

**Review by RubenRBS — APPROVED** [review]
That's a nice check to have, good idea

File: etc/scripts/util/test/cases/suspicious_path.properties (hunk 2 above)
Comment (dkm) [reply etc/scripts/util/test/cases/suspicious_path.properties]: done

**Pull request merged**